src/libraries/System.Runtime/tests/System/Exception.Helpers.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Tests
 {
-    public static class ExceptionUtility
+    public static class ExceptionHelpers
```

Comment:
The reason for this rename was due to a conflicting name with a type that exists in `Xunit` namespace that is pulled in via `xunit.runner.utility` because `XHarness.Test.Runners` depends on it: https://github.com/dotnet/xharness/blob/master/src/Microsoft.DotNet.XHarness.Tests.Runners/Microsoft.DotNet.XHarness.Tests.Runners.csproj#L15